src/main/java/com/cloudbees/jenkins/plugins/amazonecs/ECSTaskTemplate.java

```diff
@@ -528,6 +542,11 @@ public String getLabel() {
         return label;
     }
 
+    public String getAgentContainerName() {
+
```

Comment:
Nit, if you end up making changes, there's a blank line here that could be removed.

Reply:
Done.